mne/io/base.py

```diff
@@ -1565,7 +1565,8 @@ def plot_psd(self, tmin=0.0, tmax=None, fmin=0, fmax=np.inf,
                  proj=False, n_fft=2048, picks=None, ax=None,
                  color='black', area_mode='std', area_alpha=0.33,
                  n_overlap=0, dB=True, average=True, show=True,
-                 n_jobs=1, line_alpha=None, verbose=None):
+                 n_jobs=1, line_alpha=None, spatial_colors=False,
```

Comment:
We already went to ``average=True``, so deprecation cycle would mean that we would still use the old way of plotting. Unless we do ``average=True`` and throw a deprecation warning for one release cycle and then go ``average=False, spatial_colors=True`` for the next. I would suggest just going straight to ``spatial_colors=True``. Wdyt?